src/components/article.tsx

```diff
@@ -27,23 +27,25 @@ const Article = ({
   <article className="article-reader">
     <h1 className="article-reader__headline">{title}</h1>
     <TOC heading="TABLE OF CONTENTS" tableOfContents={tableOfContents} />
-    <div dangerouslySetInnerHTML={{ __html: html }} />
-    <div
-      style={{
-        display: 'flex',
-        flexWrap: 'wrap',
-        marginTop: '5rem',
-        alignItems: 'center',
-      }}
-    >
-      Contributors:
-      {authors &&
-        authors.map(
-          author => author && <AuthorLink username={author} key={author} />
-        )}
+    <div className="article-reader__inner">
+      <div dangerouslySetInnerHTML={{ __html: html }} />
+      <div
+        style={{
+          display: 'flex',
+          flexWrap: 'wrap',
+          marginTop: '5rem',
+          alignItems: 'center',
+        }}
```

Comment:
`emotion` could be adapted, and perhaps styles could be written outside of JSX? An example of `emotion` used in `Pagination` component is:

https://github.com/nodejs/nodejs.dev/blob/823224fe79bed65bd99cc5e3e2464ce321d4b4ac/src/components/pagination.tsx#L11-L17

https://github.com/nodejs/nodejs.dev/blob/823224fe79bed65bd99cc5e3e2464ce321d4b4ac/src/components/pagination.tsx#L20

Reply:
Makes sense :) I agree.